carbonio-user-management-sdk.iml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Please remember to not commit or push this kind of file, this is a module settings file created by IntelliJ, so remove it from this PR and add a line in .gitignore for the iml extension